kolibri/plugins/learn/assets/src/views/TopicsContentPage.vue

```diff
@@ -286,6 +277,36 @@
       timeSpent() {
         return this.contentPageMounted ? this.$refs.contentPage.time_spent : 0;
       },
+      back() {
+        let route = {};
+        if (this.$route) {
```

Comment:
Could instead use a [return early strategy](https://szymonkrajewski.pl/why-should-you-return-early/) here, which will let you unnest the rest of the function:

```
if (!this.$route) {
  return null;
}
```
then the rest of the function can just continue without being inside an if statement.